docs/Tutorial_Contrast_Profile2.ipynb

```diff
@@ -0,0 +1,249 @@
+{
```

Comment:
<p>Is there a particular reason behind using the name <code>v</code> here? Can we use <code>T</code> instead? Like... <code>T_pos</code> and <code>T_neg</code> ?</p><p> </p><p> </p><p>ALSO: How do we know the indices 63630, 68129, etc? This can create a confusion for readers. Reading those indices made me think that using contrast profile requires us having some prior knowledge about those indices. If that is not the case, then can you provide a brief explanation on how one can create the datasets <code>T(+)</code> and <code>T(-)</code> in a real-world problem?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/800/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<blockquote>Is there a particular reason behind using the name&nbsp;v&nbsp;here? Can we use&nbsp;T&nbsp;instead? Like...&nbsp;T_pos&nbsp;and&nbsp;T_neg&nbsp;</blockquote><p>Fixed</p>